.gitignore

```diff
@@ -1,8 +1,7 @@
 # Add any directories, files, or patterns you don't want to be tracked by version control
 .git
 node_modules
-index.d.ts
-test.d.ts
+test/**/*.d.ts
```

Comment:
Would be good to leave index.d.ts in .gitignore because this file is dynamically generated before publishing. Should someone try to link library and do fancy things with it in original source form - index.d.ts might be out of date - types would be taken from ts files directly.